library/agent/api/Event.ts

```diff
@@ -64,7 +64,19 @@ export type DetectedAttack = {
   time: number;
 };
 
-type MonitoredSinkStats = {
+export type OperationKind =
+  | "sql_op"
+  | "nosql_op"
+  | "outgoing_http_op"
+  | "fs_op"
```

Comment:
would merge fs_op and path_op, the difference is not as clear in every language

Reply:
yes, makes sense! I also thought about that